### ISSUE TITLE
semicolon at end of file not detected unless there's a newline

### ISSUE THREAD

**Issue by glenjamin** [babel/babel-eslint#2]
No lint warning
```js
module.exports = "something";
```

Lint warning about semicolon when there's no newline at EOF
```js
module.exports = "something";
```

```
semicolon.js
  1:1   error  Newline required at end of file but not found  eol-last
  1:32  error  Missing semicolon                              semi

✖ 2 problems (2 errors, 0 warnings)
```

**Comment by sebmck** [comment]
Can't seem to replicate this, I'm currently doing:

```sh
$ echo "module.exports = 'foo';" >semicolon.js
$ eslint semicolon.js
```

But I'm not getting anything.

**Comment by glenjamin** [comment]
echo adds the trailing newline by default - try this:

```sh
echo -n 'module.exports = "noeofnewline";' > semicolon.js
./node_modules/.bin/eslint semicolon.js
```

**Comment by sebmck** [comment]
Oh right, duh.

**Comment by sebmck** [comment]
#1, #3 and this issue should be fixed as of 1.0.6. Thanks for trying it out!
